pre-commit

```diff
@@ -57,13 +57,13 @@ SAVEIFS=$IFS
 IFS=$(echo -en "\n\b")
 
 if [ $CHECK_ALL ]; then
-        files_to_check=$(find $git_root -path $git_root/.git -prune -o -print)
+    files_to_check=$(find "$git_root" -path "$git_root/.git" -prune -o -print)
```

Comment:
I've added {}s around all bash variables being used in strings that didn't already have them.
Also added some "'s around a couple of vars that might contain spaces.